framework/include/utils/SlepcSupport.h

```diff
@@ -53,6 +56,26 @@ PetscErrorCode mooseSlepcEigenFormJacobianB(SNES snes, Vec x, Mat jac, Mat pc, v
 PetscErrorCode mooseSlepcEigenFormFunctionA(SNES snes, Vec x, Vec r, void * ctx);
 PetscErrorCode mooseSlepcEigenFormFunctionB(SNES snes, Vec x, Vec r, void * ctx);
 PetscErrorCode mooseSlepcEigenFormFunctionAB(SNES snes, Vec x, Vec Ax, Vec Bx, void * ctx);
+PetscErrorCode mooseSlepcStoppingTest(EPS eps,
+                                      PetscInt its,
+                                      PetscInt max_it,
+                                      PetscInt nconv,
+                                      PetscInt nev,
+                                      EPSConvergedReason * reason,
+                                      void * ctx);
+PetscErrorCode mooseSlepcEPSGetSNES(EPS eps, SNES * snes);
+PetscErrorCode mooseSlepcEPSMonitor(EPS eps,
+                                    int its,
+                                    int nconv,
+                                    PetscScalar * eigr,
+                                    PetscScalar * eigi,
+                                    PetscReal * errest,
+                                    int nest,
+                                    void * mctx);
+
+PetscErrorCode mooseSlepcEPSSNESSetUpOptionPrefix(EPS eps);
+PetscErrorCode mooseSlepcEPSSNESSetCustomizePC(EPS eps);
+PetscErrorCode mooseSlepcEPSSNESKSPSetPCSide(FEProblemBase & problem, EPS eps);
```

Comment:
It looks like have a history of not documenting these APIs, but I would love to see some doxygen here. Most of the method names are pretty self explanatory, but for instance I don't know just by looking at it what `mooseSlepcStoppingTest` does

Reply:
Will do docs